stories/App.story.js

```diff
@@ -197,48 +207,53 @@ storiesOf('Timeline', module)
   )
   .add(
     'Event Styling',
-    withInfo('TimelineEvent is completely customizable')(() =>
+    withInfo('TimelineEvent is completely customizable')(() => (
       <Timeline>
         <TimelineEvent
           title='John Doe sent a SMS'
           createdAt='2016-09-12 10:06 PM'
           icon={<i className='material-icons md-18'>textsms</i>}
           iconColor='#6fba1c'
-          style={{backgroundColor: '#fff', padding: 10, boxShadow: '0 0 3px 1px #BD3B36', border: '1px solid #eee'}}
-          contentStyle={{backgroundColor: '#00BCD4', color: '#fff'}}
+          style={{
+            backgroundColor: '#fff',
+            padding: 10,
+            boxShadow: '0 0 3px 1px #BD3B36',
+            border: '1px solid #eee'
+          }}
+          contentStyle={{ backgroundColor: '#00BCD4', color: '#fff' }}
         >
           This message should appear on a different background
         </TimelineEvent>
       </Timeline>
-    )
+    ))
   )
   .add(
     'Icon and Bubble Styling',
-    withInfo('Modify the appearance of bubbles and containing icons')(() =>
-      <Timeline lineStyle={{background: '#999', width: 3}}>
+    withInfo('Modify the appearance of bubbles and containing icons')(() => (
+      <Timeline lineStyle={{ background: '#999', width: 3 }}>
         <TimelineEvent
           title='John Doe sent a SMS'
           createdAt='2016-09-12 10:06 PM'
           icon={<i className='material-icons md-18'>textsms</i>}
           iconColor='#6fba1c'
-          bubbleStyle={{backgroundColor: '#00ff81'}}
+          bubbleStyle={{ backgroundColor: '#00ff81' }}
         >
           Notice the style variations to the bubble and icons
         </TimelineEvent>
         <TimelineEvent
           title={'Phone event'}
           createdAt='2016-10-12 12:12 AM'
           icon={<i className='material-icons md-18'>phone</i>}
-          iconStyle={{marginLeft: 1, marginTop: 0}}
+          iconStyle={{ marginLeft: 1, marginTop: 0 }}
           iconColor='#5C6BC0'
         >
           John called!
         </TimelineEvent>
       </Timeline>
-    )
+    ))
   )
   .add(
-    'Dynamic Prop Updates',
+    '[DontTest] Dynamic Prop Updates',
```

Comment:
Skip snapshot testing, because refs https://stackoverflow.com/questions/23199909/using-tolocalestring-in-node-js/23200062#23200062